MembraneVideoroomDemo.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -6,17 +6,17 @@
         "repositoryURL": "https://github.com/google/promises.git",
         "state": {
           "branch": null,
-          "revision": "ec957ccddbcc710ccc64c9dcbd4c7006fcf8b73a",
-          "version": "2.2.0"
+          "revision": "3e4e743631e86c8c70dbc6efdc7beaa6e90fd3bb",
+          "version": "2.1.1"
```

Comment:
Is this needed to remove the --allow-warnings flag? If so I don't like it. In my opinion if we want to do something about that flag it shouldn't be done by using older dependencies since it will block us from using newer standards later on.

Reply:
no, --allow-warnings has nothing to do with it, I don't know why xcode suddenly installed older versions :p I'll try to update it